UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "Name": "ForceChangeRate",
+  "BaseUnit": "NewtonPerSecond",
+  "XmlDoc": "Force change rate is the ratio of the force change to the time during which the change occurred (value of force changes per unit time).",
+  "Units": [
+    {
+      "SingularName": "NewtonPerSecond",
+      "PluralName": "NewtonsPerSecond",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
```

Comment:
We could add some of the common prefixes here, but I am not familiar with this unit and what prefixes would be commonly used.